tests/simple.cpp

```diff
@@ -94,11 +94,13 @@ TEST_CASE_METHOD(SimpleFixture, "Send/Recv Test", "[simple]")
 
   // atleast one message needs to be >2GB to test for the case of multi-piece
   // messages
-  std::size_t sizes[11] = { 288, 128, 64, 32, 64, 32, 32, 64, 32, 32 };
+  static const std::size_t sizes[] = { 288, 128, 64, 32, 64, 32, 32, 64, 32, 32 };
```

Comment:
Ah, good point about `array`, although the point about `vector` stands. But it's a very minor point. Let's not worry about it. I'll merge in the change now.